app/modules/user/lib/email_tests.js

```diff
@@ -26,3 +32,29 @@ const emailTests = {
     return !isEmail(email)
```

Comment:
Unfortunately, this would invalidate all current translations of that string, which could be prevented by manually capitalizing all translations of that string, or by procrastinating on this by using the `I18n` function, which is what I suggest we do: 444d3d58a